save-backend/src/test/kotlin/org/cqfn/save/backend/docker/ContainerManagerTest.kt

```diff
@@ -0,0 +1,44 @@
+package org.cqfn.save.backend.docker
+
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import kotlin.io.path.ExperimentalPathApi
+import kotlin.io.path.createTempFile
+
+@OptIn(ExperimentalPathApi::class)
+class ContainerManagerTest {
+    private lateinit var containerManager: ContainerManager
+    private lateinit var testContainerId: String
+
+    @BeforeEach
+    fun setUp() {
+        containerManager = if (System.getProperty("os.name").startsWith("Windows")) {
+            // for docker inside WSL2 use it's eth0 network IP
+            ContainerManager("tcp://172.20.51.70:2375")
```

Comment:
what is this IP for?

Reply:
This is IP of WLS2 network, to communicate with docker daemon inside WLS2 from project opened under Windows. At least until IDEA supports working with projects from WSL